deployment/docker/compose/sso/docker-compose.keycloak.yml

```diff
@@ -5,7 +5,7 @@ services:
     container_name: kapua-keycloak
     image: kapua/kapua-keycloak:${IMAGE_VERSION}
     ports:
-      - "9090:8080"
+      - "9090:9090"
```

Comment:
KC_PORT is configurable in the keycloak image - is there any way to share the variable here?